docs/stringvalidator/isnetwork.md

```diff
@@ -0,0 +1,62 @@
+# `IsNetwork`
+
+!!! quote inline end "Released in v1.8.0"
+
+This validator is a generic validator for checking if the string is a valid network format.
+
+Some network formats are :
+
+## How to use it
+
+The validator takes a list of `NetworkValidatorType` as argument and a boolean.
```

Comment:
The validator takes a list of `NetworkValidatorType` and a boolean as argument .

Reply:
Done